Topic1/key1/Customer.java

```diff
@@ -0,0 +1,122 @@
+package Topic1.key1;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.ArrayList;
+
+import Topic1.key1.EcommerceWebPage.payMethod;
+
+public class Customer {
```

Comment:
The responsability of Customer object is not clear. You can pay, checkout and even search products with a Customer. I would suggest to rename it to CustomerAccount or CustomerProfile and move non customer related behavior to an appropiate new or existing object.